app/views/posts/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_with model:@post, html: { class: 'post-form'} do |form|%>
+    <h1>Create Post</h1>
+    <%=form.label :title%>
+    <%=form.text_field :title, class: 'post-text-field'%>
+    <%=form.label "Post"%>
+    <%=form.text_area :text, class: 'post-text-area'%>
+    <%=form.submit "Post", class: 'common-btn'%>
+    <%end%> 
```

Comment:
- [ ] Kindly fix the error that comes when a user calls the new post route(new view, new action for the post). Your app doesn't have a button to access the post new view, so I just typed the URL for the post new action.

![new](https://github.com/jeanbuhendwa/blog_app/assets/53124548/58b300df-c978-4980-972b-e0df2a5326fa)